app/lib/dqt/params/trn_request.rb

```diff
@@ -0,0 +1,116 @@
+# frozen_string_literal: true
+
+module Dqt
+  module Params
+    class TrnRequest
+      UNITED_KINGDOM = "United Kingdom"
+
+      GENDER_CODES = {
+        male: "Male",
+        female: "Female",
+        other: "Other",
+        gender_not_provided: "Other",
+      }.freeze
+
+      PROGRAMME_TYPE = {
+        TRAINING_ROUTE_ENUMS[:assessment_only] => "AssessmentOnlyRoute",
+        TRAINING_ROUTE_ENUMS[:early_years_assessment_only] => "EYITTAssessmentOnly",
+        TRAINING_ROUTE_ENUMS[:early_years_postgrad] => "EYITTGraduateEntry",
+        TRAINING_ROUTE_ENUMS[:early_years_salaried] => "EYITTGraduateEmploymentBased",
+        TRAINING_ROUTE_ENUMS[:early_years_undergrad] => "EYITTUndergraduate",
+        TRAINING_ROUTE_ENUMS[:hpitt_postgrad] => "TeachFirstProgramme",
+        TRAINING_ROUTE_ENUMS[:opt_in_undergrad] => "UndergraduateOptIn",
+        TRAINING_ROUTE_ENUMS[:pg_teaching_apprenticeship] => "Apprenticeship",
+        TRAINING_ROUTE_ENUMS[:provider_led_postgrad] => "Core",
+        TRAINING_ROUTE_ENUMS[:provider_led_undergrad] => "Core",
+        TRAINING_ROUTE_ENUMS[:school_direct_salaried] => "SchoolDirectTrainingProgrammeSalaried",
+        TRAINING_ROUTE_ENUMS[:school_direct_tuition_fee] => "SchoolDirectTrainingProgrammeSelfFunded",
+      }.freeze
+
+      DEGREE_CLASSES = {
+        Dttp::CodeSets::Grades::FIRST_CLASS_HONOURS => "FirstClassHonours",
+        Dttp::CodeSets::Grades::UPPER_SECOND_CLASS_HONOURS => "UpperSecondClassHonours",
+        Dttp::CodeSets::Grades::LOWER_SECOND_CLASS_HONOURS => "LowerSecondClassHonours",
+        Dttp::CodeSets::Grades::THIRD_CLASS_HONOURS => "ThirdClassHonours",
+        Dttp::CodeSets::Grades::OTHER => "GeneralDegree",
+        Dttp::CodeSets::Grades::PASS => "Pass",
+      }.freeze
```

Comment:
Most grades have a sensible mapping, except for "Other", which I've marked as "GeneralDegree".

Other possible values:
```
UndividedSecondClassHonours
FourthClassHonours
UnclassifiedHonours
Aegrotat
PassFollowingAnHonoursCourse
Ordinary
Distinction
Merit
Notapplicable
NotKnown
HigherDegree
```

Reply:
Checked with Rob C - mapping 'Other' to 'NotKnown' is preferred here.